### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by JuvitaSaini09**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+## BuildUI
+
+BuildUI is a component Library built using HTML and CSS.You can use Component Library in your projects With the help of Component Library ,you don't need to code from scratch,You just pick component from library Which ever you need and start using it .
+
+## Usage /Installation ##
+
+First,you need to import the links in this page :
+https://build-ui-component-library.netlify.app/index.html
+
+Import hese links in your file to use Component Library
```
Comment (Muzammil-khan710) [inline README.md]: Typo

File: README.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+## How to run app ##
+
+To use any Component,you just first need to import the above link in you file and then just copy and paste the HTML of the components.
```
Comment (Muzammil-khan710) [inline README.md]: Typo

**Review by Muzammil-khan710 — APPROVED** [review]
Readme is good and very well readable. There are some typos have a look there.However it is good to merge.  For now i am approving this PR

File: README.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+## 4. Button ##
+
+Buttons are main component of any website.They are highly used component.
+
+We have 5 buttons
+
+• Primary Buttons
+• Link Buttons
+• Icon Buttons
+• Floating Buttons
```
Comment (Sahil25061999) [inline README.md]: typo probably, you have mentioned 5 buttons

**Review by Sahil25061999 — APPROVED** [review]
Looks good to me, just a suggestion you can link the component to their respective section.

File: README.md (hunk 3 above)
Comment (JuvitaSaini09) [reply README.md]: fixed

File: README.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+• White Alert
+• Danger Alert
+• Warning Alert
+• Success Alert
+
+## 2. Avatar ##
+
+Avatars are images that users can set as their profile picture.
+
+We have 4 types of avatar
```
Comment (gauravsinhaweb) [inline README.md:60]: I would recommend instead of 'we' use the brand name `BuildUI` or `it` (optional).

File: README.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+
+## 8. Text-utilites ##
+
+Text of variant styles and sizes can be used according to their respective class names.
+
+We have 4 types of text
+
+•  Headings
+•  Small text
+• Grey text
```
Comment (gauravsinhaweb) [inline README.md:128]: not aligned with sibling texts

File: README.md [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+
+## 10. Navigation-Desktop ##
+
+Navigation Bar contain simply list of links which helps us to navigate to website.Here is as simple desktop navgation bar.
+
+We have one navigation bar component
+
+## 11. Modal ##
+
+Add dialogs to your site for lightboxes, user notifications, or completely custom content.We have one Modal component
```
Comment (gauravsinhaweb) [inline README.md:148]: missed the full stop.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+
+Toasts are lightweight notifications designed to mimic the push notifications that have been popularized by mobile and desktop operating systems. They’re built with flexbox, so they’re easy to align and position.
+
+## 14. Simplified Grid ##
+
+A grid layout provides a versatile layout to place elements on the page.There are two grid layouts available two and three column layouts.
+
+We have two types of grid
+
+• Simple List:ordered and unordered
```
Comment (gauravsinhaweb) [inline README.md:166]: spacing after `list:`.

**Review by gauravsinhaweb — COMMENTED** [review]
Readme looks well organized. the listing makes it more readable and clean. Good to merge after minor changes.

File: README.md (hunk 7 above)
Comment (JuvitaSaini09) [reply README.md:166]: updated,thanks for review

File: README.md (hunk 6 above)
Comment (JuvitaSaini09) [reply README.md:148]: fixed

File: README.md (hunk 5 above)
Comment (JuvitaSaini09) [reply README.md:128]: fixed

File: README.md (hunk 4 above)
Comment (JuvitaSaini09) [reply README.md:60]: I have updated it, thanks for the review

**Pull request merged**